.github/workflows/build.yml

```diff
@@ -63,6 +62,8 @@ jobs:
               -DCOREAUDIO=ON
               -DHSS1394=ON
               -DMACOS_BUNDLE=ON
+              -DMODPLUG=OFF
+              -DWAVPACK=OFF
```

Comment:
Why are you turning these off?

Reply:
Not present in the macOS buildenv

Reply:
I thought modplug is? I don't think wavpak is though.

Reply:
Just checked. Both are absent in the macOS build environment.